feature/staff/src/main/java/io/github/droidkaigi/confsched2020/staff/ui/viewmodel/StaffsViewModel.kt

```diff
@@ -27,32 +37,41 @@ class StaffsViewModel @Inject constructor(
         }
     }
 
-    private val staffContentsLoadState: LiveData<LoadState<StaffContents>> = liveData {
-        emitSource(
-            staffRepository.staffs()
-                .toLoadingState()
-                .asLiveData()
+    private val store = StoreBuilder.fromNonFlow<String, StaffResponse> { api.getStaffs() }
```

Comment:
Thanks!
How about providing this instance in the repository dagger module 👀

Reply:
dependency at repository module is not resolved, so i inject at StaffModule...

Reply:
> Thanks!
> How about providing this instance in the repository dagger module

How about that??